scipy/sparse/linalg/isolve/iterative.py

```diff
@@ -93,12 +94,14 @@ def _get_atol(tol, atol, bnrm2, get_residual, routine_name):
     """
 
     if atol is None:
+        '''
```

Comment:
unwanted change here too

Reply:
Indeed, it is a redundant comment and I'll remove it, thanks